core/src/main/scala/kafka/server/QuotaFactory.scala

```diff
@@ -55,19 +56,25 @@ object QuotaFactory {
     )
   }
 
-  def clientProduceConfig(cfg: KafkaConfig): ClientQuotaManagerConfig =
+  def clientProduceConfig(cfg: KafkaConfig): ClientQuotaManagerConfig = {
+  if (cfg.producerQuotaBytesPerSecondDefault != Long.MaxValue)
+    warn(s"${KafkaConfig.ProducerQuotaBytesPerSecondDefaultProp} has been deprecated in 0.11.0 and will be removed in a future release. Use dynamic quota defaults instead.")
     ClientQuotaManagerConfig(
       quotaBytesPerSecondDefault = cfg.producerQuotaBytesPerSecondDefault,
       numQuotaSamples = cfg.numQuotaSamples,
       quotaWindowSizeSeconds = cfg.quotaWindowSizeSeconds
     )
+  }
 
-  def clientFetchConfig(cfg: KafkaConfig): ClientQuotaManagerConfig =
+  def clientFetchConfig(cfg: KafkaConfig): ClientQuotaManagerConfig = {
+    if (cfg.consumerQuotaBytesPerSecondDefault != Long.MaxValue)
+      warn(s"${KafkaConfig.ConsumerQuotaBytesPerSecondDefaultProp} has been deprecated in 0.11.0 and will be removed in a future release. Use dynamic quota defaults instead.")
```

Comment:
@ijuma Updated. Thank you for the review. I will merge to trunk and 0.11.0 branch when the build completes.